### PR TITLE
Write accessor min/max conditionally

### LINKED ISSUE #325: Reduce accessor min/max size in JSON
Current output includes min/max for every accessor, and writes them with high precision:

```
      "max": [
        0.9957213401794434,
        0.9957212805747986,
        0.9957213401794434
      ],
      "min": [
        9.9999988758398e-13,
        9.9999988758398e-13,
        9.9999988758398e-13
      ],
```

This takes up more space than necessary in the JSON output. We could omit min/max for many accessor types, and truncate precision somewhat where we do include it.

### PR DESCRIPTION
**Pull request by donmccurdy**
Fixes #325.

### CODE REVIEW

File: packages/core/src/io/writer-context.ts [hunk 1]
```diff
@@ -111,9 +111,19 @@ export class WriterContext {
...
-		accessor.getMax((accessorDef.max = []));
-		accessor.getMin((accessorDef.min = []));
-		accessorDef.normalized = accessor.getNormalized();
+
+		const needsBounds = this._doc.getGraph()
+			.listParentLinks(accessor)
+			.some((link) => link.getName() === 'POSITION');
+		if (needsBounds) {
+			accessor.getMax((accessorDef.max = []).map(Math.fround));
+			accessor.getMin((accessorDef.min = []).map(Math.fround));
```
Comment (lexaknyazev) [inline packages/core/src/io/writer-context.ts:120]: If `accessor.getMax` fetches values from binary buffers, then `Math.fround` is a no-op.

Comment (donmccurdy) [reply packages/core/src/io/writer-context.ts:120]: Yes, missed this! Fixed in https://github.com/donmccurdy/glTF-Transform/commit/dedd34c1b78986a214b4730149f3c29799146bd4.

**Comment by donmccurdy** [comment]
Cleanup + bug fix in https://github.com/donmccurdy/glTF-Transform/commit/dedd34c1b78986a214b4730149f3c29799146bd4.

**Pull request merged**